README.md

```diff
@@ -122,6 +121,27 @@ If you don't want to display comments for a particular post you can disable them
 
 --
 
+### Social networks
+
+You can add links to the accounts you have on other sites, with respective icon, by adding one or more of the following options in your config:
+
+```yaml
+twitter_username: jekyllrb
+github_username:  jekyll
+dribbble_username: dribbble
+facebook_username: fb
+flickr_username: flickr
+instagram_username: ig
+linkedin_username: linkedin
+pinterest_username: pinterest
+stackoverflow_username: stackoverflow
```

Comment:
since this is documentation, I think we'll have to point to the right way as well.
e.g. StackOverflow profiles are located at `users/` i.e. for `username: minima`, the profile is actually at `stackoverflow.com/users/[random]/minima`